gateway/cli/src/main.rs

```diff
@@ -21,8 +21,8 @@ use ln_gateway::rpc::{
 };
 use serde::Serialize;
 
-const DEFAULT_WAIT_FOR_CHAIN_SYNC_RETRIES: u32 = 12;
-const DEFAULT_WAIT_FOR_CHAIN_SYNC_RETRY_DELAY_SECONDS: u64 = 10;
+const DEFAULT_WAIT_FOR_CHAIN_SYNC_RETRIES: u32 = 60;
```

Comment:
Might be helpful for `wait_for_chain_sync` to have a `debug!` log or something to tell if it ever gets stuck

Reply:
`poll` already had it - another reason not to remove(?)